dist/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
What do you think about using file-loader and html webpack plugin in order to generate html file and load images?